src/migrations/20210304115810-change-default-timestamp-to-now.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.up = function(db, cb) {
+    db.runSql(
+        `
+        ALTER TABLE client_applications ALTER COLUMN created_at SET DEFAULT now();
```

Comment:
Should now() be enclosed in quotes like in the down migration?

Reply:
Nope, that's the whole point of the migration, enclosing it in quotes makes postgres run `now()` at query time, hence setting the default to whatever `now()` resolves to at the time of the migration as a constant. Without quotes tells postgres to run the function on insert if the field is unset, therefore making it dynamic.

Reply:
Thanks for the clarification.